[2024-01-17T13:03:18.217+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sql_example.create_update_dim_customer manual__2024-01-17T13:03:05.170453+00:00 [queued]>
[2024-01-17T13:03:18.264+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sql_example.create_update_dim_customer manual__2024-01-17T13:03:05.170453+00:00 [queued]>
[2024-01-17T13:03:18.268+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 0
[2024-01-17T13:03:18.371+0000] {taskinstance.py:2192} INFO - Executing <Task(BigQueryExecuteQueryOperator): create_update_dim_customer> on 2024-01-17 13:03:05.170453+00:00
[2024-01-17T13:03:18.420+0000] {standard_task_runner.py:60} INFO - Started process 3332 to run task
[2024-01-17T13:03:18.460+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sql_example', 'create_update_dim_customer', 'manual__2024-01-17T13:03:05.170453+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/data_transform_BQ.py', '--cfg-path', '/tmp/tmp_hq9__xc']
[2024-01-17T13:03:18.490+0000] {standard_task_runner.py:88} INFO - Job 33: Subtask create_update_dim_customer
[2024-01-17T13:03:18.857+0000] {task_command.py:423} INFO - Running <TaskInstance: sql_example.create_update_dim_customer manual__2024-01-17T13:03:05.170453+00:00 [running]> on host fafdf46f827b
[2024-01-17T13:03:19.953+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='phungquan' AIRFLOW_CTX_DAG_ID='sql_example' AIRFLOW_CTX_TASK_ID='create_update_dim_customer' AIRFLOW_CTX_EXECUTION_DATE='2024-01-17T13:03:05.170453+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-17T13:03:05.170453+00:00'
[2024-01-17T13:03:19.968+0000] {bigquery.py:1245} INFO - Executing: CREATE OR REPLACE TABLE 
  `OLAP.Dim_Customers` AS
SELECT
  customers.CustomerID,
  customers.City,
  customers.Phone,
  customers.Country
FROM
  `OLTP.Customers` AS customers
[2024-01-17T13:03:20.033+0000] {connection.py:234} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-01-17T13:03:20.047+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-01-17T13:03:20.115+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/google/cloud/hooks/bigquery.py:2109 AirflowProviderDeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2024-01-17T13:03:20.120+0000] {credentials_provider.py:353} INFO - Getting connection using `google.auth.default()` since no explicit credentials are provided.
[2024-01-17T13:03:20.457+0000] {bigquery.py:1596} INFO - Inserting job ***_1705496600443942_110dc7cdc8723b206594a2ec75de8f20
[2024-01-17T13:03:24.206+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=sql_example, task_id=create_update_dim_customer, execution_date=20240117T130305, start_date=20240117T130318, end_date=20240117T130324
[2024-01-17T13:03:24.526+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-17T13:03:24.844+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
